kbase-extension/static/kbase/js/common/sdk.js

```diff
@@ -104,15 +92,10 @@ define(['common/props'], (Props) => {
                  * TODO: more cases of bad default value? Or a generic
                  * default value validator?
                  */
-                if (!defaultValues || defaultValues.length === 0) {
```

Comment:
`defaultValues` is set a couple of lines earlier so it's never going to be false